src/bivariate_normals.jl

```diff
@@ -126,14 +126,18 @@ immutable BvnComponent{NumType <: Number}
       c = 1 ./ (the_det^.5 * 2pi)
       major_sd = sqrt(maximum([ the_cov[1, 1], the_cov[2, 2] ]))
 
+      precision = Array(NumType, 2, 2)
+      precision[2, 2] = the_cov[1,1] / the_det
+      precision[1, 1] = the_cov[2,2] / the_det
+      precision[1, 2] = -the_cov[1, 2] / the_det
+      precision[2, 1] = -the_cov[2, 1] / the_det
+
       if calculate_siginv_deriv
         # Derivatives of Sigma^{-1} with respect to sigma.  These are the second
         # derivatives of log|Sigma| with respect to sigma.
         # dsiginv_dsig[a, b] is the derivative of sig^{-1}[a] / d sig[b]
         dsiginv_dsig = Array(ThisNumType, 3, 3)
 
-        precision = the_cov^-1
```

Comment:
My branch would probably require https://github.com/JuliaArrays/StaticArrays.jl/pull/51 depending on which functions in Celeste that are exercised. I'll try to finish up the PR to `StaticArrays` now.